dotCMS/src/main/java/com/dotcms/publisher/bundle/business/BundleAPI.java

```diff
@@ -130,19 +130,19 @@
 	 *
 	 * @param olderThan {@link Date} will remove all sent bundles older than it
 	 * @param user      {@link User} User to check the deleting permissions
-	 * @return Set of bundle ids deleted
+	 * @return BundleDeleteResult, Set of bundle ids deleted
 	 * @throws	DotDataException	thrown when an error in the underlying data layer occurs
 	 */
-	Set<String> deleteBundleAndDependenciesOlderThan(Date olderThan, User user) throws DotDataException;
+	BundleDeleteResult deleteBundleAndDependenciesOlderThan(Date olderThan, User user) throws DotDataException;
```

Comment:
Is BundleAPI a new interface? Because the method signature has changed. Be aware as this change might break customers codebase

Reply:
this is a new method that was created on the previous PR of this same issue